client/src/components/Header/Header.js

```diff
@@ -53,8 +51,10 @@ export default function Home() {
       </Grid>
       <Grid container item xs={6} justify="flex-end">
         <Grid item>
-          <Button href={'/login'}>
-            <h4>Login</h4>
+          <Button>
+            <Link to='/login' className={classes.link}>
```

Comment:
Lol, this is what I was talking about the conflicting changes because of your code and Jason's code touching the same lines. This is fine this time because of the changes aren't vast. No need to fix it again, we'll address it during the session where we go to do the merge.